### DISCUSSION TITLE
How to limit QSO search to one log?

### DISCUSSION THREAD

**Discussion by R1BLH** [magicbug/Cloudlog#2980] [General]
When I search for a QSO, I get results from all my logs. How do I limit the search to the active log?
![изображение](https://github.com/magicbug/Cloudlog/assets/16667173/504b818e-6fa9-42cd-ac49-cec69b048ee0)
In the screenshot the results from two logs -- R1BLH (active), R900BL (not active, special event station).

It look like the same reason as in  #2946.

**Comment by magicbug** [comment]
At the minute you can't but seems a reasonable thing to offer as a toggled choice for the user.

Issue created and its on the dev list
